src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/ArcaneMagnet.java

```diff
@@ -0,0 +1,56 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.magical;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.items.settings.DoubleRangeSetting;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import io.github.thebusybiscuit.slimefun4.implementation.items.blocks.UnplaceableBlock;
+import org.bukkit.entity.ExperienceOrb;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+/**
+ * The {@link ArcaneMagnet} is a {@link SlimefunItem} that allows a {@link Player} to
+ * automatically pick up {@link Item items} and {@link ExperienceOrb experience} in a certain radius while holding shift and having an
+ * {@link ArcaneMagnet} in their {@link Inventory}.
+ *
+ * @author TheBusyBiscuit
```

Comment:
did cookie work on this?

Reply:
I put cookie under because the code was based off of the original code for the infused magnet